src/App.tsx

```diff
@@ -10,33 +11,26 @@ export const tabs = [
 ];
 
 export const App: React.FC = () => {
+  const [selectedTabId, setSelectedTabId] = useState('tab-1');
```

Comment:
Use the value from the `tabs` variable here. And use the whole tab in this state, not only id
```suggestion
  const [selectedTabId, setSelectedTabId] = useState(tabs[0]);
```

Reply:
done